src/mscorlib/shared/System/ReadOnlySpan.cs

```diff
@@ -125,8 +125,8 @@ internal ReadOnlySpan(ref T ptr, int length)
             _length = length;
         }
 
-        //Debugger Display = {T[length]}
-        private string DebuggerDisplay => string.Format("{{{0}[{1}]}}", typeof(T).Name, _length);
+        //Debugger Display = System.ReadOnlySpan<T>[length]
+        private string DebuggerDisplay => string.Format("System.ReadOnlySpan<{0}>[{1}]", typeof(T).Name, _length);
```

Comment:
We should not need this anymore now that Span has overriden ToString method.

Debugger calls overriden ToString method by default to get the display string. See https://msdn.microsoft.com/en-us/library/x810d419.aspx

Reply:
> We should not need this anymore now that Span has overriden ToString method.

Great. I didn't realize that.

> If a class has an overridden ToString() method, the debugger uses the overridden method instead of the default {<typeName>}. Thus, if you have overridden the ToString() method, the debugger uses the overridden method instead of the default{<typeName>}, and you do not have to use DebuggerDisplay. If you use both, the DebuggerDisplay attribute takes precedence over the overridden ToString() method.

Reply:
Could there be any issues with the DebuggerDisplay if we make the proposed changes to ToString() from https://github.com/dotnet/corefx/issues/26584#issuecomment-360615501?

Reply:
Right, it should have the desired behavior. You presumably want to see `Span<char>` as a string in the debugger window.

Reply:
Cool, good to know!